platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/MapboxMapOptions.java

```diff
@@ -76,6 +75,8 @@
 
   private String style;
 
+  private float pixelRatio;
```

Comment:
Since we are adding an option to set the pixel ratio, I think this value should also be pushed to the `UiSettings` during map initialisation.